ast/term.go

```diff
@@ -822,7 +822,7 @@ func (v Var) IsWildcard() bool {
 
 // IsGenerated returns true if this variable was generated during compilation.
 func (v Var) IsGenerated() bool {
-	return strings.HasPrefix(string(v), "__local")
+	return strings.HasPrefix(string(v), "__local") || strings.HasPrefix(string(v), "__qlocal")
```

Comment:
:thinking: if we make make it a suffix then we don't have to perform two prefix checks. I'd consider this since `IsGenerated()` is called all over the place. `__localq<id>__`?

Reply:
Its so simple 😆 I'll update to that

Reply:
Done.